manifests/params.pp

```diff
@@ -145,22 +151,24 @@
         }
       }
 
-      $service_name       = 'elasticsearch'
-      $service_hasrestart = true
-      $service_hasstatus  = true
-      $service_pattern    = $service_name
-      $defaults_location  = '/etc/sysconfig'
-      $pid_dir            = '/var/run/elasticsearch'
     }
     'Debian', 'Ubuntu': {
       $service_name       = 'elasticsearch'
       $service_hasrestart = true
       $service_hasstatus  = true
       $service_pattern    = $service_name
-      $service_providers  = 'init'
       $defaults_location  = '/etc/default'
-      $init_template      = 'elasticsearch.Debian.erb'
-      $pid_dir            = false
+
+      if (($::operatingsystem == 'Debian' and $::operatingsystemmajrelease >= 8) or
```

Comment:
Like i mentioned i would love to see this more verbose instead of a one liner.
My best idea would be to have Debian and Ubuntu have their own spot in the case `$::operatingsystem` case part and then do a check for the `$::operatingsystemmajrelease` value.

thanks!